products/urls.py

```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from products.views import ProductDetailView
+
+urlpatterns = [
+    path("/product/<id>", ProductDetailView.as_view()),
```

Comment:
```suggestion
    path("/product/<int:product_id>", ProductDetailView.as_view()),
```
- data type을 정확하게 정해줴요.
- 어떤 리소스의 id인지 명확하게 해주는 것이 좋습니다.

Reply:
넵 변경했습니다